src/scss/_vars.scss

```diff
@@ -145,14 +180,29 @@ $display-size4: var(--heading-size4);
 
 // HEADING
 $heading-size1: var(--heading-size1);
+$heading-line-height-size1: var(--heading-line-height-size1);
```

Comment:
Is there a reason we're adding line height tokens? Wouldn't heading1 always have the same line height associated with it? Do we have a need to mix and match tokens?

Reply:
we had the same line height before, and now we have different for each case. so we have to do that